apps/web/components/shared/project-card-variants.tsx

```
@@ -1,384 +1,384 @@
-                strokeWidth="2"
-                strokeLinecap="round"
-                strokeLinejoin="round"
-            />
-        </svg>
-    )
+	const UserAvatar = () => (
+		<svg
+			viewBox="0 0 24 24"
+			fill="none"
+			className="w-full h-full p-1 text-gray-400"
+		>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)